client/js/ifml/elements/event/index.js

```diff
@@ -46,16 +46,18 @@ exports.Event = joint.shapes.basic.Generic.extend({
         type: 'ifml.Event',
         size: {width: 20, height: 20},
         name: {text: 'event', vertical: 'top', horizontal: 'left-outer'},
+        stereotype: 'user',
         fixedParent: true,
         attrs: {
             '.': {magnet: 'passive'},
             '.ifml-event-magnet-rect': {magnet: true, visibility: 'hidden'},
+            '.ifml-event-system': {visibility: 'hidden'},
```

Comment:
Probably `ifml-event-system-symbol` would be more descriptive if we add other stereotypes in the future